compass/ocean/tests/utility/create_salin_restoring/__init__.py

```diff
@@ -0,0 +1,28 @@
+from compass.ocean.tests.utility.create_salin_restoring.extrap_salin import (
+    ExtrapSalin,
+)
+from compass.ocean.tests.utility.create_salin_restoring.\
```

Comment:
@xylar I'm stuck on this line.  If I use this form, it passes flake8 but fails isort.  If I use the previous form 
```
from compass.ocean.tests.utility.create_salin_restoring.salinity_restoring import (
    Salinity,
)
```
It fails flake8 but passes isort.  I'm at a loss on what to do

Reply:
Sometimes, you just need to have a long line, e.g.:
https://github.com/MPAS-Dev/compass/blob/7333744570d0ac174ae626192034aa2bf609b839/compass/landice/tests/ismip6_forcing/ocean_basal/__init__.py#L4

```
from compass.ocean.tests.utility.create_salin_restoring.salinity_restoring import (  # noqa: E501
    Salinity,
)  
```

Reply:
In general, a comment with `  # noqa: <CODE>` will bypass the linting on that line for that issue.

Reply:
Thanks @xylar for that tip!